.rubocop.yml

```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
All disableling of cops should be in this file. If a cop is disabled, it means it's not part of the project's style guidelines.